examples/acados_python/casadi_tests/test_casadi_LICQ_violation.py

```diff
@@ -0,0 +1,96 @@
+import sys
+sys.path.insert(0, '../getting_started')
+import numpy as np
+import casadi as ca
+from typing import Union
+
+from acados_template import AcadosOcp, AcadosOcpSolver, AcadosCasadiOcpSolver
+from pendulum_model import export_pendulum_ode_model
+
+def formulate_ocp(Tf: float = 1.0, N: int = 20)-> AcadosOcp:
+    ocp = AcadosOcp()
+
+    # set model
+    model = export_pendulum_ode_model()
+    ocp.model = model
+
+    nx = model.x.rows()
+    nu = model.u.rows()
+    
+    # set prediction horizon
+    ocp.solver_options.N_horizon = N
+    ocp.solver_options.tf = Tf
+
+    # cost matrices
+    Q_mat = 2*np.diag([1e3, 1e3, 1e-2, 1e-2])
+    R_mat = 2*np.diag([1e-2])
+
+    # path cost
+    ocp.cost.cost_type = 'NONLINEAR_LS'
+    ocp.model.cost_y_expr = ca.vertcat(model.x, model.u)
+    ocp.cost.yref = np.zeros((nx+nu,))
+    ocp.cost.W = ca.diagcat(Q_mat, R_mat).full()
+
+    # terminal cost
+    ocp.cost.cost_type_e = 'NONLINEAR_LS'
+    ocp.cost.yref_e = np.zeros((nx,))
+    ocp.model.cost_y_expr_e = model.x
+    ocp.cost.W_e = Q_mat
+
+    # set constraints
+    Fmax = 80
+    ocp.constraints.lbu = np.array([-Fmax])
+    ocp.constraints.ubu = np.array([+Fmax])
+    ocp.constraints.idxbu = np.array([0])
+
+    ocp.constraints.x0 = np.array([0, np.pi, 0, 0])  # initial state
+    ocp.constraints.idxbx_0 = np.array([0, 1, 2, 3])
```

Comment:
I don't think it makes sense to set `idxbx_0` here, as it is already done when setting `x0`.